packages/libro-lab/src/editor-viewer/module.ts

```diff
@@ -14,4 +16,6 @@ export const CodeEditorViewerModule = ManaModule.create(
   CodeEditorViewerOpenHandler,
   LibroDefaultViewerOpenHandler,
   JSONLanguageSpecs,
+  RLanguageSpecs,
```

Comment:
Do we have a default way to open text files?

Reply:
the part you quoted is used to handle highlighting.